app/question.js

```diff
@@ -48,8 +53,8 @@ define(['jquery', 'app/messages', 'app/config', 'app/string'], function ($, mess
   }
 
   function showProgress() {
-    $( "#progress-number" ).html( "<p>" + (totalLength - items.length) + "/" + totalLength + " words</p>" );
-    var percentageVal = percentage(totalLength - items.length, totalLength);
+    $( "#progress-number" ).html( "<p>" + itemsAnsweredCorrectly + "/" + totalLength + " words</p>" );
```

Comment:
Since we will have internationalization module, these messages will be moved to language files